styles/styles.css

```diff
@@ -0,0 +1,807 @@
+/* ======= LINK FOR GOOGLE FONTS ======= */
+@import url('https://fonts.googleapis.com/css2?family=Dela+Gothic+One&family=Lato:ital,wght@0,300;1,400&display=swap');
+
+/* ======= GLOBAL STYLES RULES ======= */
+:root {
+  /* === COLORS === */
+  --color-dark: #272a31;
+  --color-darkOrange: #ec5242;
+  --color-grey: #d3d3d3;
+  --color-white: #fff;
+  --color-whitesmoke: #f9f9f9;
+
+  /* === FONTS FAMILY === */
+  --font-family-1: 'Lato', sans-serif;
+  --font-family-2: 'Dela Gothic One', cursive;
```

Comment:
- [ ] I highly advise you to only use the given fonts as it creates discrepancies in relation to the original design. So Kindly replace the `--font-family-2` with `cocogoose`.
- [ ] And even though you've declared the "Lato" font here most of the texts in your page are using the browser's default font. 
- here's how it currently looks:
 ![firefox_XaZH4BML07](https://github.com/lily4178993/leaders-week/assets/104968236/17e7a6ee-ea70-49ba-9f90-9d50bcadcc42)